include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -247,95 +230,164 @@ struct TypeInfo;
 struct Value;
 struct ModelMetadata;
 
+/** \brief The Env (Environment)
+*
+* The Env holds the logging state used by all other objects.
+* <b>Note:</b> One Env must be created before using any other Onnxruntime functionality
+*/
 struct Env : Base<OrtEnv> {
-  Env(std::nullptr_t) {}
+  explicit Env(std::nullptr_t) {} ///< \brief Create an empty Env object, must be assigned a valid one to be used 
```

Comment:
>Create [](http://example.com/codeflow?start=46&length=6)

general - can we be consistent with the verb form used to describe a method? e.g., "create" vs. "creates"

Reply:
We can try. Depends on if we're saying what the function does or what it's used for. It's used to create, but what it does is that it creates.

Reply:
"Depends on if we're saying what the function does or what it's used for."
consistently do one or the other would be good